### ISSUE TITLE
Support batch size > 1

### ISSUE THREAD

**Issue by annaproxy** [Sasafrass/straattaal#22]
Current code (and my PR) can only use batch size = 1.

A larger batch size would improve processing speed and make the final models better.

There are probably some torch or torchtext tools to help you deal with varying data point sizes that we could use. (Or just manually pad everything and discard all padding)

**Comment by Sasafrass** [comment]
Can't we use _Pad Packed Sequence_ https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pad_packed_sequence.html and its inverse _Pack Padded Sequence_ https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pack_padded_sequence.html for this purpose?

**Comment by annaproxy** [comment]
It looks like those functions are now native and not terribly slow anymore. I will give it a shot soon.

**Comment by Sasafrass** [comment]
oo right, they used to be ridiculously slow. Good catch.
https://github.com/Sasafrass/straattaal/blob/528df42f45ee38944703d11873735377a38b1bac/requirements.txt#L31 should already be on torch 1.9 - Always in favor of upgrading to any latest stable release.
I have created an issue for this (https://github.com/Sasafrass/straattaal/blob/master/environment.yml) and also added it to the newly created Dependency Management board (https://github.com/Sasafrass/straattaal/projects/7)
